lib/active_merchant/billing/gateways/pago_facil.rb

```diff
@@ -117,6 +117,14 @@ def json_error(response)
           "raw_response" => response
         }
       end
+
+      def country_code(country)
+        if country
+          country = ActiveMerchant::Country.find(country)
+          country.to_s
+        end
+      rescue InvalidCountryCodeError
```

Comment:
I don't think we should be swallowing this error. Shouldn't the caller know if they're passing in bad country data?